hack/print-workspace-status.sh

```diff
@@ -80,4 +80,6 @@ GIT_VERSION ${GIT_VERSION-}
 GIT_BRANCH ${GIT_BRANCH-}
 GIT_RELEASE_TAG ${GIT_RELEASE_TAG-}
 GIT_RELEASE_COMMIT ${GIT_RELEASE_COMMIT-}
+AWS_ACCESS_KEY_ID ${AWS_ACCESS_KEY_ID-}
+AWS_SECRET_ACCESS_KEY ${AWS_SECRET_ACCESS_KEY-}
```

Comment:
Are these here only for current test/debugging purposes?

Reply:
There's not very many ways to get external variables into Bazel unfortunately :( This was the best solution I've found.

Reply:
https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/573/files#r257046530 applies here as well.

Reply:
Is the output of `hack/print-workspace-status.sh` shown in the logs that are uploaded for viewing in test-grid?